### PR TITLE
Have at least three sites in each state

### PR DESCRIPTION
**Pull request by stucka**
as in 343 and 346

### CODE REVIEW

File: newshomepages/sources/sites.csv [hunk 1]
```diff
@@ -588,7 +594,7 @@ YCityNewsOnline,https://ycitynews.com/,Y-City News,Zanesville,America/New_York,U
 YahooFinance,https://finance.yahoo.com/,Yahoo Finance,New York City,America/New_York,US,en,,
 YahooNews,https://news.yahoo.com/,Yahoo! News,New York City,America/New_York,US,en,us-national,
 YediotAhronot,https://www.yediot.co.il/,Yedioth Ahronoth,Tel Aviv,Asia/Tel_Aviv,IL,he,,
-fiquemsabendo,https://fiquemsabendo.substack.com/,Fiquem Sabendo,Sao Paulo,America/Sao_Paulo,BR,pt,,
+_fiquemsabendo,https://fiquemsabendo.substack.com/,Fiquem Sabendo,Sao Paulo,America/Sao_Paulo,BR,pt,,
```
Comment (palewire) [inline newshomepages/sources/sites.csv:597]: We cut the leading `_` on this solve an archive.org bug. Can you catch up with the tweak in main?

**Review by palewire — CHANGES_REQUESTED** [review]

**Comment by palewire** [comment]
Are you sure all of the bundles you're adding are in the bundles.csv file?

**Comment by stucka** [comment]
Leading underscore fixed. All sites' bundles verified. I'll Slack you my lil helper script.

File: newshomepages/sources/sites.csv [hunk 2]
```diff
@@ -1062,7 +1073,7 @@ uwogazette,https://westerngazette.ca/,Western Gazette,London,America/New_York,CA
 valoreconomico,https://valor.globo.com/,Valor Econômico,Sao Paulo,America/Sao_Paulo,BR,pt,,
 vancouversun,https://vancouversun.com/,Vancouver Sun,Vancouver,America/Vancouver,CA,en,,
 vcstar,https://www.vcstar.com/,Ventura County Star,Camarillo,America/Los_Angeles,US,en,socal|gannett|california,
-verge,https://www.theverge.com/,Verge,New York City,America/New_York,US,en,tech,
```
Comment (palewire) [inline newshomepages/sources/sites.csv:1065]: Did we lose The Verge?

**Review by palewire — CHANGES_REQUESTED** [review]

**Comment by palewire** [comment]
Gotcha. Verge thing is my only remaining question.

**Review by stucka — COMMENTED** [review]
Bring back verge

**Comment by palewire** [comment]
Thank you!

**Pull request merged**